src/components/scenes/TransactionDetailsScene.tsx

```diff
@@ -560,10 +560,10 @@ export const TransactionDetailsScene = withWallet((props: OwnProps) => {
   if (swapData != null && typeof swapData.payoutCurrencyCode === 'string') {
     swapData.payoutCurrencyCode = swapData.payoutCurrencyCode.toUpperCase()
   }
+  const destinationWallet = useSelector(state => (swapData ? state.ui.wallets.byId[swapData.payoutWalletId] : undefined))
   const destinationDenomination = useSelector(state =>
-    swapData ? getDisplayDenomination(state, wallet.currencyInfo.pluginId, swapData.payoutCurrencyCode) : undefined
+    swapData && destinationWallet != null ? getDisplayDenomination(state, destinationWallet.pluginId, swapData.payoutCurrencyCode) : undefined
```

Comment:
Can we do `swapData != null` to make these two lines more explicit?

Reply:
Sure can